grafana_client/elements/plugin.py

```diff
@@ -46,17 +43,26 @@ def uninstall_plugin(self, pluginId):
             r = self.client.POST(path)
             return r
         except Exception as ex:
-            self.logger.info("Skipped uninstalling %s and error = %s", pluginId, ex)
+            if errors == "raise":
+                raise
+            elif errors == "ignore":
+                self.logger.info(f"Skipped uninstalling plugin {pluginId}: {ex}")
+            else:
+                raise ValueError(f"error={errors} is invalid")
         return None
 
+    def health_check_plugin(self, pluginId):
+        """
+        :return:
+        """
+        path = "/plugins/%s/health" % pluginId
+        r = self.client.GET(path)
+        return r
+
     def get_plugin_metrics(self, pluginId):
         """
         : return:
         """
-        try:
-            path = "/plugins/%s/metrics" % pluginId
-            r = self.client.GET(path)
-            return r
-        except Exception as ex:
-            self.logger.info("Got error in fetching metrics for plugin %s and error = %s", pluginId, ex)
-        return None
+        path = "/plugins/%s/metrics" % pluginId
```

Comment:
Thanks for refactoring. I should have put more focus here since that was my local logging to help identify if plugin does not have metrics endpoint.